scripts/pylint_extensions.py

```diff
@@ -1464,6 +1464,45 @@ def process_module(self, node):
                 line=closing_line_index_of_fileoverview + 1)
 
 
+class BlankLineBelowDocstring(checkers.BaseChecker):
+    """Add lint check to ensure that there is one blank newline
+    below each class docstring.
+    """
+
+    __implements__ = interfaces.IRawChecker
+    name = 'space_between_imports_and_file-overview'
+    priority = -1
+    msgs = {
+        'C0026': (
+            'Please provide an empty line below each calss docstring.'
+        )
+    }
+
+    def process_module(self, node):
+        """Process a module to ensure that there is a blank line below
+        each class docstring.
+        """
+
+        multi_line_indicator = b'"""'
+        file_content = read_from_node(node)
+        file_length = len(file_content)
+        have_a_class = False
+        expect_empty_line = False
+        for line_num in python_utils.RANGE(file_length):
+            line = file_content[line_num].strip()
+            if expect_empty_line:
+                expect_empty_line = False
+                have_a_class = False
+                if not file_content[empty_line_check_index + 1] == b'\n':
```

Comment:
what is `empty_line_check_index` here?

Reply:
Also, add a check if there is more than one blank line.

Reply:
Hi @Hudda !
I put it by mistake, it was supposed to be line_num. I added the check, I will add test for that after I fix the errors. 
Thanks!